### ISSUE TITLE
distort_image_ops_test.py fails on tf-nightly-2.0-gpu

### ISSUE THREAD

**Issue by WindQAQ** [tensorflow/addons#373]
The CI Build for PY2-GPU failed because of `distort_image_ops_test.py`.
https://source.cloud.google.com/results/invocations/7ed10b8d-6cba-47d9-b764-d823365528f4/log

Ideally, the manager should catch the exception raised in CC file
https://github.com/tensorflow/addons/blob/436580c5427af1b0666faf4955cd493f5ccace72/tensorflow_addons/image/distort_image_ops_test.py#L110-L112

https://github.com/tensorflow/addons/blob/436580c5427af1b0666faf4955cd493f5ccace72/tensorflow_addons/custom_ops/image/cc/kernels/adjust_hsv_in_yiq_op.cc#L70-L73

**Comment by seanpmorgan** [comment]
So more info:
Seems not related to py2 vs py3. There has not been a gpu-py3 released since 20190718 so thats why it's passing.

The error is incredibly creptic when decorated with tf.function... and when I remove that it looks like 2 errors are being raised instead of the one we're catching:

```
InvalidArgumentError: input must be at least 3-D, got shape[2,3] [Op:AdjustHsvInYiq]
```
and
```
ValueError: Shape must be at least rank 3 but is rank 2 for 'adjust_hsv_in_yiq/AdjustHsvInYiq' (op: 'AdjustHsvInYiq') with input shapes: [2,3], [], [], [].
```

I have no idea which commit started this and no reasons why it's only happening on the GPU pip package.

**Comment by feihugis** [comment]
It can run in the eager mode but failed in the graph mode. I tried to add the GPU kernel but the build failed. Will update it here if I get any progress.

**Comment by seanpmorgan** [comment]
For tracking purposes... a temporary fix was put in #378
